_layouts/base.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 ---
+{% assign keywords = "google,asylo,opensource framework,flexible framework,opensource application,asylo-site" %}
```

Comment:
I admit I'm not a Jekyll expert. Does this not override values set in the page front matter? I expected default values (that can be overridden by front matter) to be declared something similar to https://jekyllrb.com/docs/configuration/front-matter-defaults/

Reply:
need to remove from this and add it to the front matter defaults (ie to the index.html)

Reply:
Do you have a reference/documentation for that being the case? It doesn't seem to work. With your current PR contents, all pages except for index.html have empty keywords.